### PR TITLE
[unittest] added zoneout mask unittest

### PR DESCRIPTION
**Pull request by lhs8928**
- Test the zoneout mask is generated according to the zoneout rate

Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

### CODE REVIEW

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: test/unittest/unittest_nntrainer_tensor.cpp [hunk 1]
```diff
@@ -3610,6 +3610,137 @@ TEST(nntrainer_Tensor, cat_02_n) {
...
+    unsigned int zeros = 0;
+    unsigned int ones = 0;
+    for (unsigned int i = 0; i < opposite.size(); ++i) {
+      if (is_near(opposite.getValue(i), 0.0f)) {
+        ++zeros;
+      } else if (is_near(opposite.getValue(i), 1.0f)) {
+        ++ones;
+      } else {
+        // This should not be happen
+        EXPECT_EQ(false, true);
```
Comment (jihochu) [inline test/unittest/unittest_nntrainer_tensor.cpp:3667]: How about using `FAIL()` ?

Comment (lhs8928) [reply test/unittest/unittest_nntrainer_tensor.cpp]: Thanks for your notice :)

**Review by jihochu — APPROVED** [review]
LGTM

**Pull request merged**